config/core/resources/pingsource.yaml

```diff
@@ -333,7 +481,7 @@ spec:
   conversion:
     strategy: Webhook
     webhook:
-      conversionReviewVersions: ["v1", "v1beta1"]
+      conversionReviewVersions: ["v1", "v1beta2", "v1beta1"]
```

Comment:
this is the `ConversionReview` version. Remove `v1beta2`

Reply:
removed.